huxley/logging/mail.py

```diff
@@ -21,7 +23,18 @@ def send_messages(self, email_messages):
             for email in email_messages:
                 log_email(email)
 
-                return super(LoggingEmailBackend, self)._send(email)
+                try:
+                    return super(LoggingEmailBackend, self)._send(email)
+                except SMTPException:
+                    logger = logging.getLogger('huxley.api')
+                    exc_type, exc_value, exc_traceback = sys.exc_info()
+                    exc_traceback = ''.join(traceback.format_exception(exc_type, exc_value, exc_traceback))
+                    log = json.dumps({
+                          'message': exc_traceback,
+                          'uri': ', '.join(email.to),
+                          'status_code': 500,
+                          'username': ''})
```

Comment:
Is there any way we'd have access to the current user in this method? Would be useful information to have.